app_dart/lib/src/foundation/utils.dart

```diff
@@ -104,6 +106,24 @@ Future<List<LuciBuilder>> getRepoBuilders(HttpClientProvider branchHttpClientPro
       .toList();
 }
 
+/// First gets luci try builders from any commit.
+///
+/// Returns luci try builders by filtering `run_if` property over changed files.
+Future<List<LuciBuilder>> getLuciTryBuilders(
```

Comment:
It's confusing to see both `getLuciBuilders` and `getLuciTryBuilders`, especially `getLuciBuilders` can return try builders as well. Consider merging these two functions.

Reply:
Yeah, thought about this earlier, combining logics would add the complexity to the function. But as you pointed out, combining should clear out any confusion. Just combined logics to a single function `getLuciBuilders`.